src/components/privateRoute/PrivateRoute.js

```diff
@@ -19,7 +19,7 @@ const PrivateRoute = ({ component: Component, ...props }) => {
 };
 
 PrivateRoute.propTypes = {
-  component: PropTypes.node.isRequired
+  component: PropTypes.func.isRequired
```

Comment:
You can destructure the propTypes right in the import like this:
```js
import { func } from 'prop-types';
```
And then:
```suggestion
  component: func.isRequired
```